pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -3265,6 +3265,19 @@ type CephBlockPoolRadosNamespaceList struct {
 	Items           []CephBlockPoolRadosNamespace `json:"items"`
 }
 
+// CephBlockPoolRadosNamespaceSpecMirroring represents the mirroring configuration of CephBlockPoolRadosNamespaceSpec
+type CephBlockPoolRadosNamespaceSpecMirroring struct {
```

Comment:
`CephBlockPoolRadosNamespaceSpecMirroring` to CephBlockPoolRadosNamespaceMirroringSpec`

Reply:
Types within other CRDs don't need the full naming convention. How about a shorter name such as this?
```suggestion
type RadosNamespaceMirroring struct {
```